eng/versioning/version_client.txt

```diff
@@ -62,14 +62,50 @@ com.azure:azure-storage-internal-avro;12.0.0;12.0.1-beta.3
+com.azure.spring:azure-spring-boot-starter-servicebus-jms;3.0.0-beta.1;3.0.0-beta.1
+com.azure.spring:azure-spring-boot-starter;3.0.0-beta.1;3.0.0-beta.1
+com.azure.spring:azure-spring-boot;3.0.0-beta.1;3.0.0-beta.1
+com.azure.spring:azure-spring-cloud-autoconfigure;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-context;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-eventhubs-stream-binder;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-messaging;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-servicebus-queue-stream-binder;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-servicebus-stream-binder-core;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-servicebus-topic-stream-binder;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-cache;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-eventhubs-kafka;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-eventhubs;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-servicebus;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-starter-storage-queue;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-storage;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-stream-binder-test;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-cloud-telemetry;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-data-gremlin;2.3.1-beta.1;2.3.1-beta.1
+com.azure.spring:azure-spring-integration-core;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-integration-eventhubs;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-integration-servicebus;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-integration-storage-queue;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-integration-test;2.0.0-beta.1;2.0.0-beta.1
+com.azure.spring:azure-spring-starter-storage;2.0.0-beta.1;2.0.0-beta.1
+com.microsoft.azure:spring-cloud-azure-appconfiguration-config-web;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-azure-feature-management-web;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-azure-feature-management;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
+com.azure.spring:azure-spring-boot-test-aad;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-test-core;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-test-keyvault;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-test-servicebus-jms;1.0.0;1.0.0
+com.azure.spring:azure-spring-cloud-test-eventhubs;1.0.0;1.0.0
+com.azure.spring:azure-spring-cloud-test-storage;1.0.0;1.0.0
+com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config-test;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
This test module should also be changed its group ID and artifact ID, but we could do it another PR.